examples/tabs/js/alt.js

```diff
@@ -0,0 +1,4 @@
+var Alt = require('../../../src/alt')
+var alt = new Alt()
+
+module.exports = alt
```

Comment:
:+1: preference it is for example and docs.  Though it should be consistent within an example or docs page.